microsoft/testsuites/hpc/infinibandsuite.py

```diff
@@ -31,6 +32,33 @@
     """,
 )
 class InfinibandSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. List all available network interfaces
+        2. Check if InfiniBand cards are present
+        3. Ensure InfiniBand cards are named starting with "ib0"
+        """,
+        priority=2,
+        requirement=simple_requirement(supported_features=[Infiniband]),
+    )
+    def verify_ib_naming(self, log: Logger, node: Node) -> None:
+        infiniband = node.features[Infiniband]
+        ib_interfaces = infiniband.get_ib_interfaces()
+        ib_first_device_name = ib_interfaces[0].ib_device_name
+    
+        r = re.compile("ib[0-9]")
+        ib_names = list(filter(r.match, ib_first_device_name))
+        if len(ib_names) == 0:
+            raise UnsupportedDistroException(
```

Comment:
Do you have a list of unsupported distros? If so, we can skip the test for those distros since those are not testable but this error indicates failure.

Reply:
Modified to be LISAException.